apps/impulse/android/app/build.gradle

```diff
@@ -22,6 +22,12 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
+ def keystoreProperties = new Properties()
+ def keystorePropertiesFile = rootProject.file('key.properties')
```

Comment:
Should this file be available in the project? Does this file contain our singing key? Please add a comment that this fails on dev machines and is an intended behavior.

Reply:
It's not intended to check in our key. So this is required.
I don't really see where this results in a failure on dev...
While developing, you're not building the app with the release target.